internal/apiserver/route_get_identity_by_id.go

```diff
@@ -20,17 +20,19 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/google/uuid"
 	"github.com/hyperledger/firefly-common/pkg/ffapi"
+	"github.com/hyperledger/firefly-common/pkg/i18n"
 	"github.com/hyperledger/firefly/internal/coremsgs"
 	"github.com/hyperledger/firefly/pkg/core"
 )
 
 var getIdentityByID = &ffapi.Route{
 	Name:   "getIdentityByID",
-	Path:   "identities/{iid}",
+	Path:   "identities/{id:.+}",
```

Comment:
I'm not sure you need the `.+` if you are verifying the path parameter underneath

Reply:
This helps the API interpret the `/` character in the DID string properly, without this the API will return a 404 error code. I will add this comment to the code.

Reply:
aaah yeah makes sense